scripts/profiler/rtt_nordic_config.py

```diff
@@ -5,9 +5,13 @@
 
 RttNordicConfig = {
     'device_snr' : None,
-    'rtt_info_channel': 2,
-    'rtt_data_channel': 1,
-    'rtt_command_channel': 1,
+    'rtt_up_channel_names' :  {
+        'Nordic profiler info' : 'info',
+        'Nordic profiler data' : 'data',
+    },
+    'rtt_down_channel_names' : {
+        'Nordic profiler command' : 'command',
```

Comment:
Why are there a space in front of the `:` ?
```suggestion
    'rtt_up_channel_names':  {
        'Nordic profiler info': 'info',
        'Nordic profiler data': 'data',
    },
    'rtt_down_channel_names': {
        'Nordic profiler command': 'command',
```

Reply:
fixed